projects/1.20/assets/better-archeology/betterarcheology/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "itemGroup.betterarcheology": "更好的考古学",
+
+  "block.betterarcheology.rotten_log": "腐朽的原木",
+  "block.betterarcheology.rotten_planks": "腐朽的木板",
+  "block.betterarcheology.rotten_slab": "腐朽的台阶",
+  "block.betterarcheology.rotten_stairs": "腐朽的楼梯",
+  "block.betterarcheology.rotten_fence": "腐朽的围栏",
+  "block.betterarcheology.rotten_fence_gate": "腐朽的围栏门",

```

```suggestion
  "block.betterarcheology.rotten_fence": "腐朽的栅栏",
  "block.betterarcheology.rotten_fence_gate": "腐朽的栅栏门",
```
参见原版术语`栅栏`/`Fence`以及`栅栏门`/`Fence Gate`